### PR TITLE
Updated Rezact CLI app.

### PR DESCRIPTION
**Pull request by neghi14**
Hi Zach, I refactored the CLI application to be prompt based as it's easier to work with than remembering specific flags, although i intend to add flags later for who may want it.  I added both a typescript and tailwind template for people that want that option, I also updated the Rezact default homepage in the templates. do give it a look and tell me what you think. Thank you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@rezact/create-app",
-  "version": "1.0.9",
+  "version": "1.5.0",
```
Comment (zachlankton) [inline package.json]: Excellent Work! Can you adjust this to 1.1.0 ?

I believe these changes merits a single minor bump in versioning, unless you have a compelling argument for the larger jump?

**Review by zachlankton — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (neghi14) [reply package.json]: Ok Zach, I'll do that immediately

**Comment by neghi14** [comment]
Adjusted to version 1.1.0, please do take a look when you can.

**Review by zachlankton — APPROVED** [review]
Excellent Work!!

**Pull request merged**